test/integration/api/rest_api_test.go

```diff
@@ -25,6 +26,9 @@ func TestRestApi(t *testing.T) {
 		t.Skip("An integration test")
 	}
 
+	err := os.Setenv("TRIVY_VERSION", "0.1.6")
```

Comment:
This will cause side effects for any following tests that get run after this as it's not cleared/cleaned up.